diff --git a/Makefile.am b/Makefile.am
index 8af1864..cc6ea80 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -19,7 +19,7 @@ EXTRA_DIST = \
 	build-aux/missing
 applicationsdir = $(datadir)/applications
 applications_DATA = htop.desktop
-pixmapdir = $(datadir)/pixmaps
+pixmapdir = $(datadir)/icons/hicolor/128x128/apps
 pixmap_DATA = htop.png
 appicondir = $(datadir)/icons/hicolor/scalable/apps
 appicon_DATA = htop.svg
