diff --git a/CommandLine.c b/CommandLine.c
index d21f482..3d97608 100644
--- a/CommandLine.c
+++ b/CommandLine.c
@@ -118,6 +118,7 @@ static CommandLineStatus parseArguments(const char* program, int argc, char** ar
       {"no-unicode", no_argument,         0, 'U'},
       {"tree",       no_argument,         0, 't'},
       {"pid",        required_argument,   0, 'p'},
+      {"alt-keys",   no_argument,         0, 'a'},
       {"filter",     required_argument,   0, 'F'},
       {"highlight-changes", optional_argument, 0, 'H'},
       {"readonly",   no_argument,         0, 128},
@@ -131,6 +132,9 @@ static CommandLineStatus parseArguments(const char* program, int argc, char** ar
       if (opt == EOF)
          break;
       switch (opt) {
+         case 'a':
+            FunctionBar_useAltKeys(true);
+            break;
          case 'h':
             printHelpFlag(program);
             return STATUS_OK_EXIT;
diff --git a/FunctionBar.c b/FunctionBar.c
index 0850037..9c0e056 100644
--- a/FunctionBar.c
+++ b/FunctionBar.c
@@ -17,6 +17,11 @@ in the source distribution for its full text.
 #include "ProvideCurses.h"
 #include "XUtils.h"
 
+static bool AltKeys = false;
+static const char* Alt_FunctionBar_FKeys[] = {" h", " S", " /", " \\", " t", " >", " ]", " [", " k", " q", NULL};
+void FunctionBar_useAltKeys(bool use_alt_keys) {
+   AltKeys = use_alt_keys;
+}
 
 static const char* const FunctionBar_FKeys[] = {"F1", "F2", "F3", "F4", "F5", "F6", "F7", "F8", "F9", "F10", NULL};
 
@@ -56,7 +61,7 @@ FunctionBar* FunctionBar_new(const char* const* functions, const char* const* ke
       this->size = i;
    } else {
       this->staticData = true;
-      this->keys.constKeys = FunctionBar_FKeys;
+      this->keys.constKeys = AltKeys ? Alt_FunctionBar_FKeys : FunctionBar_FKeys;
       this->events = FunctionBar_FEvents;
       this->size = ARRAYSIZE(FunctionBar_FEvents);
    }
diff --git a/FunctionBar.h b/FunctionBar.h
index f01a5ef..2ca77a0 100644
--- a/FunctionBar.h
+++ b/FunctionBar.h
@@ -21,6 +21,8 @@ typedef struct FunctionBar_ {
    bool staticData;
 } FunctionBar;
 
+void FunctionBar_useAltKeys(bool use_alt_keys);
+
 FunctionBar* FunctionBar_newEnterEsc(const char* enter, const char* esc);
 
 FunctionBar* FunctionBar_new(const char* const* functions, const char* const* keys, const int* events);
